Example/index.ios.js

```diff
@@ -0,0 +1,242 @@
+import React, { Component } from 'react';
```

Comment:
Do you know offhand the differences between android and ios?  They should hopefully be close to parity.

Reply:
I replicated your index.js on index.android.js and index.ios.js as the index.js file was not found

Reply:
That's weird.  It should find it if it's called index.js.

Reply:
That was what i thought, but it was unable to find it, so i had to change it

Reply:
@slycoder can you approve the review?

Reply:
I'd really like to get to the bottom of this first.  If you're in a pinch, can you revert the index.js change, I'll merge it, and then try to repro locally and we can address the issue in a separate PR.

Reply:
This issue of `index.android` not found happens on certain android devices when `getJSMainModuleName()` is not defined, like defined in master now: https://github.com/blackuy/react-native-twilio-video-webrtc/blob/master/Example/android/app/src/main/java/com/example/MainApplication.java#L32